src/Effects.js

```diff
@@ -357,7 +338,9 @@ module.exports = class Effects {
           const pizza = this.pizza[i];
           p5.translate(pizza.x, pizza.y);
           p5.rotate(pizza.rot);
+          p5.scale(2, 2);
           drawPizza(p5._renderer.drawingContext);
+          p5.pop();
```

Comment:
We've now got one `push()` and two `pop()` calls in each of these draw methods, which seems like a bug.  Can you explain why `pop()` is needed here when it also exists below (line 350)?

Reply:
will address these throughout. thanks!